apps/bajour/pages/mitmachen.tsx

```diff
@@ -3,28 +3,47 @@ import {ApiV1, AuthTokenStorageKey, SubscribeContainer} from '@wepublish/website
 import {setCookie} from 'cookies-next'
 import {NextPageContext} from 'next'
 import getConfig from 'next/config'
+import {useRouter} from 'next/router'
 
 import {Container} from '../src/components/layout/container'
 
 export default function Mitmachen() {
   const locationOrigin = typeof window !== 'undefined' ? location.origin : ''
+  const {
```

Comment:
@michael-scheurer the default query parameters

Reply:
https://wepublish.atlassian.net/wiki/spaces/WC/pages/1131249665/Publisher+Documentation#About-upselling-%26-url-parameters